client/app/admin/admin.service.ts

```diff
@@ -0,0 +1,22 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable, throwError as observableThrowError } from 'rxjs';
+import { catchError } from 'rxjs/operators';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class AdminService {
+
+    constructor(
+        private http: HttpClient
+    ) { }
+
+    getCounter(): Observable<any> {
+
```

Comment:
Empty line not needed here. 

the link to the endpoint could just be `/api/admin` and the proxy will convert it for you.